npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "marathon-ui",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
We already updated the version number in the package.json, but this is the first update of the npm-shrinkwrap.